packages/web-components/fast-foundation/src/slider/slider.ts

```diff
@@ -496,9 +490,7 @@ export class FASTSlider extends FormAssociatedSlider implements SliderConfigurat
             window[eventAction]("mousemove", this.handleMouseMove);
 
             if (e) {
-                e.preventDefault();
```

Comment:
Do we need to remove this?

Reply:
Yes this is preventing the focus naturally going to `slider`